src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt

```diff
@@ -31,5 +37,13 @@ class MigrationQueueService(
     )
   }
 
+  // given counts are approximations there is only a probable chance this returns the correct result
+  fun isItProbableThatThereAreStillMessagesToBeProcessed(): Boolean =
+    migrationSqsClient.countMessagesOnQueue(migrationQueueUrl) > 0
+
```

Comment:
decided to ignore messages in progress since worse case scenario is it just about to finish (and this message itself would be in-progress)